### ISSUE TITLE
[Bug] Error on upload: 'The language does not exist !'

### ISSUE THREAD

**Issue by henningBunk** [cianru/huawei-publish-gradle-plugin#2]
Hi there,
thanks for the plugin. I got two apps. One supports multiple languages including Russian and I had no problems uploading that app with this plugin. The other app only supports German and is only whitelisted for German users in the App Gallery. 

When I try to upload this app I get the error:

```
Huawei AppGallery Publishing API: Found apk file: `my-app-release.apk`
Huawei AppGallery Publishing API: Get Credentials
Huawei AppGallery Publishing API: Get Access Token
Huawei AppGallery Publishing API: Get App ID
Huawei AppGallery Publishing API: Get Upload Url
Huawei AppGallery Publishing API: Upload APK '/path/to/my/app.apk'
Huawei AppGallery Publishing API: Update App File Info

> Task :app:publishHuaweiAppGalleryMyAppRelease FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:publishHuaweiAppGalleryMyAppRelease'.
> Update App File Info is failed. Response: UpdateAppFileInfoResponse(ret=Ret(code=204144650, msg=[AppGalleryConnectPublishService]The language does not exist !))
```

I am not certain what the problem is but I suspect the hardcoded `lang="ru-RU"` query paramter in [HuaweiServiceImpl](https://github.com/cianru/huawei-publish-gradle-plugin/blob/master/src/main/kotlin/ru/cian/huawei/publish/service/HuaweiServiceImpl.kt#L143).

Could this be the problem? Would it help not the send the language since it's [optional for apk files](https://developer.huawei.com/consumer/en/doc/development/AppGallery-connect-References/agcapi-app-file-info_v2)? Alternativly it could be possible to set the language via gradle.

**Comment by nqnhat1988** [comment]
I got the same problem, tried to change the lang to supported one, it worked. @henningBunk

**Comment by frmz** [comment]
Its an easy fix, please .)

**Comment by cosic** [comment]
Hi @henningBunk, @nqnhat1988 . I deleted language parameter. Could you reproduce your problem on `1.0.2-SNAPSHOT` version?

**Comment by henningBunk** [comment]
For now, Huawei publishing is on hold for me so I wont be able to test this soon. Sorry.

**Comment by MindaugasVarkalys** [comment]
I have the same issue but it looks like the current snapshot version does not even do the upload, since most the code is commented out. It just logs some information. Can you update the snapshot version with the working one?

**Comment by cosic** [comment]
@MindaugasVarkalys I've uploaded the new `1.0.2-SNAPSHOT` version. Check it please.

**Comment by MindaugasVarkalys** [comment]
@cosic I still experience the same issue. I don't think you've uploaded the new version. Nexus Repository Manager says it was updated on September 15th.

![image](https://user-images.githubusercontent.com/6805516/93707440-46ee0b00-fb37-11ea-85ad-885c69c4fbff.png)

**Comment by cosic** [comment]
@MindaugasVarkalys Sorry, you right. I'd uploaded wrong version. Please check `1.0.2-SNAPSHOT` again.

**Comment by MindaugasVarkalys** [comment]
Great! It works now!

**Comment by cosic** [comment]
Fixed at v1.1.0
